Formula/orientdb.rb

```diff
@@ -48,6 +43,16 @@ def post_install
     (var/"log/orientdb").mkpath
     touch "#{var}/log/orientdb/orientdb.err"
     touch "#{var}/log/orientdb/orientdb.log"
+
+    # set the server's root user  password to changeme
+    ENV["ORIENTDB_ROOT_PASSWORD"] = "changeme"
```

Comment:
I see. Can it be blank (and bind to localhost) by default? If not, yeh `orientdb` sounds good and using a `caveats` block to output it and `post_install` to configure it is best.